atomics/T1105/T1105.yaml

```diff
@@ -298,4 +298,29 @@ atomic_tests:
       del C:\\svchost.exe >nul 2>&1
     name: command_prompt
     elevation_required: true
-
+- name: MpCmdRun DownloadFile
+  description: LOLB to download a file using Windows Defender that will put Atomic-license.txt in temp directory using this version 4.18.2007.8-0
+  supported_platforms:
+  - windows
+  input_arguments:
+    local_path:
+      description: Local path to place file
+      type: String
+      default: '%temp%\Atomic-license.txt'
+    remote_file:
+      description: URL of file to copy
+      type: url
+      default: https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/LICENSE.txt
+  dependency_executor_name: powershell
+  dependencies:
+  - description: Must have Specific Versions and this just tests for one of them
```

Comment:
Would like to see an input argument added where users can specify one of the supported versions. Then this text could be updated to says "Must have one of the Windows Defender versions: x, y, or z; Use input argument to specify the supported version you have installed"